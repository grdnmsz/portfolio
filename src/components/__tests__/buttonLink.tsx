import React from 'react'
import renderer from 'react-test-renderer'

import { ButtonLink } from '../'

describe('ButtonLink', () => {
  const props = {
    uri: '/path',
    text: 'clic',
  }

  it('should render correctly', () => {
    const tree = renderer.create(<ButtonLink {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
