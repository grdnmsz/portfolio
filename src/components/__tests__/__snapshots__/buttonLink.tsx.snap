// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLink should render correctly 1`] = `
<div
  className="rounded shadow"
>
  <a
    className="w-full flex items-center justify-center px-8 py-3 text-base font-medium rounded bg-headerColor hover:bg-teal-700 md:py-4 md:text-lg md:px-10"
    href="/path"
  >
    clic
  </a>
</div>
`;
